python/py-filelock/Portfile

```diff
@@ -17,18 +17,29 @@ long_description    This package contains a single module, which implements \
                     a platform independent file lock in Python, which \
                     provides a simple way of inter-process communication:
 
-homepage            https://filelock.readthedocs.io/en/latest/
+homepage            https://py-filelock.readthedocs.io/en/latest/api.html
```

Comment:
can this just point to `https://py-filelock.readthedocs.io` instead of the api page?